### PR TITLE
fix(UI): Error 404 when adding remote server

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This fixes a 404 error when finalising the addition of a remote server

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x
- [x] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

- Using the Wizard, add a remote server
- -> In the last step, the remote server is correctly added

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by bdauria — DISMISSED** [review]

File: www/front_src/src/PollerWizard/api/endpoints.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const legacyBaseEndpoint = './api/internal.php';
+
+export const pollerWaitListEndpoint = `${legacyBaseEndpoint}?object=centreon_configuration_remote&action=getPollerWaitList`;
+export const getPollersEndpoint = `${legacyBaseEndpoint}?object=centreon_configuration_remote&action=getRemotesList`;
+export const remoteServerWaitListEndpoint = `${legacyBaseEndpoint}?object=centreon_configuration_remote&action=getWaitList`;
+export const getRemoteServersEndpoint = `${legacyBaseEndpoint}?object=centreon_configuration_remote&action=getRemotesList`;
+export const wizardFormEndpoint = `${legacyBaseEndpoint}?object=centreon_configuration_remote&action=linkCentreonRemoteServer`;
+export const exportTaskEndpoint = `${legacyBaseEndpoint}?object=centreon_task_service&action=getTaskStatus`;
```
Comment (jdelpierre) [inline www/front_src/src/PollerWizard/api/endpoints.ts:10]:
```suggestion
const objectConfigurationRemoteUriParameter = 'object=centreon_configuration_remote';

export const pollerWaitListEndpoint = `${legacyBaseEndpoint}?${objectConfigurationRemoteUriParameter}&action=getPollerWaitList`;
export const getPollersEndpoint = `${legacyBaseEndpoint}?${objectConfigurationRemoteUriParameter}&action=getRemotesList`;
export const remoteServerWaitListEndpoint = `${legacyBaseEndpoint}?${objectConfigurationRemoteUriParameter}&action=getWaitList`;
export const getRemoteServersEndpoint = `${legacyBaseEndpoint}?${objectConfigurationRemoteUriParameter}&action=getRemotesList`;
export const wizardFormEndpoint = `${legacyBaseEndpoint}?${objectConfigurationRemoteUriParameter}&action=linkCentreonRemoteServer`;
export const exportTaskEndpoint = `${legacyBaseEndpoint}?object=centreon_task_service&action=getTaskStatus`;
```

Comment (jdelpierre) [reply www/front_src/src/PollerWizard/api/endpoints.ts:10]: too late ...

**Review by jdelpierre — APPROVED** [review]

**Review by bdauria — APPROVED** [review]

**Review by Jordan91 — APPROVED** [review]
You added the old paths to the API, just that? 
I don’t see in which particular case there was a problem. It was always present or on specific remote versions (older than API V2)?

**Comment by Thebarda** [comment]
> You added the old paths to the API, just that? I don’t see in which particular case there was a problem. It was always present or on specific remote versions (older than API V2)?



> You added the old paths to the API, just that? I don’t see in which particular case there was a problem. It was always present or on specific remote versions (older than API V2)?

The last step of the wizard called the wrong endpoint to finalise the addition of a Remote Server. It called `internal.php...` instead of `./api/internal.php....`

**Pull request merged**